libc/stdio/stdio_file.c

```diff
@@ -374,11 +374,11 @@ TEST(stdio_getput, getsputs_basic)
 
 IGNORE_TEST(stdio_getput, getsputs_readonly)
 {
-	/* 
-	Upon successful completion, fputc() shall return the value it has written. 
-	Otherwise, it shall return EOF, the error indicator for the stream shall be set 
-	and errno shall be set to indicate the error. 
-	
```

Comment:
Thanks for those fixes, however it should be done in the separate commit as it's not related to malloc asserts, maybe: `libc: remove unnecessary whitespaces`